qubesmanager/settings.py

```diff
@@ -627,7 +627,15 @@ def check_mem_changes(self):
             self.warn_too_much_mem_label.setVisible(True)
 
     def check_warn_templatenetvm(self):
-        if self.vm.klass == 'TemplateVM':
+        if self.vm.klass != 'TemplateVM':
+            return
+
+        current_netvm = self.netVM.currentData()
+        if current_netvm is None:
+            return
+
+        default_netvm = self.vm.property_get_default('netvm')
```

Comment:
```suggestion
```

Reply:
Ah yes!! Good catch. Thanks 😄

Reply:
You’re welcome!  Thanks for the contribution.